bot/cogs/core/database.py

```diff
@@ -42,6 +42,9 @@ async def populate_caches(self):
     async def fetch_user_reminder_count(self, user_id: int) -> int:
         return await self.db.fetchval("SELECT COUNT(*) FROM reminders WHERE user_id = $1", user_id)
 
+    async def fetch_user_reminders(self, user_id: int) -> list:
```

Comment:
Yeah it's returning a list, but a list of what?

https://mypy.readthedocs.io/en/stable/generics.html#defining-generic-classes